druzhba/avro.py

```diff
@@ -30,8 +31,22 @@ def _avro_format(inp):
     return inp
 
 
+def _redshift_format(inp):
+    if isinstance(inp, str):
+        # The copy command already has TRUNCATECOLUMNS turned on which will do this for us 
+        # however this protects us from the case where a columns value is ridiciously large 
+        # preventing the row from being ingested.
+        return inp[:MAX_VARCHAR_SIZE]
+    return inp
+
+
+def _format_value(inp):
+    formatted_for_redshift = _redshift_format(inp)
```

Comment:
Edge case, for sure, but I think it's probably better to have this after the `_avro_format` since it's possible for that to increase the `str` length with the Unicode decomposition.